src/properties.cpp

```diff
@@ -143,7 +143,7 @@ bool compare(const morphio::range<const morphio::Point>& vec1,
         return false;
     }
 
-    const float epsilon = 1e-6f;
+    const floatType epsilon = static_cast<morphio::floatType>(1e-6);
```

Comment:
One thing I'm wondering about these conversions… if the default type is double, are resulting floats bit-identical. Same for the reverse scenario? What about not doing these casts? Compiler warnings that are over-zealous?

Reply:
Yes, needed to keep compilers happy. better than adding macros. I will test with user-defined literals

Reply:
Uh, that sounds like a good idea! Keep in mind that we're handicapped by the f'in wheel-building process, though.